src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+import Duke.task.Deadlines;
+import Duke.task.EmptyTaskException;
+import Duke.task.Events;
+import Duke.task.Task;
+import Duke.task.ToDos;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public static ArrayList<Task> taskList;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greeting = "    ____________________________________________________________\n" +
+                "     Hello! I'm Duke\n" +
+                "     What can I do for you?\n" +
+                "    ____________________________________________________________\n";
+        String bye = "     Bye. Hope to see you again soon!\n" +
+                "    ____________________________________________________________";
+        String line = "    ____________________________________________________________\n";
+        String command = "null";
+        String spaces = "    ";
+
+        taskList = new ArrayList<>();
+
+        Scanner in = new Scanner(System.in);
+
+        FileClass file = new FileClass("data/tasks.txt");
+
+        System.out.println(greeting);
+        while (!command.equals("bye")) {
+            command = in.nextLine();
+            if (command.equals("bye")) {
+                System.out.println(line + bye);
+            } else if(command.equals("list")){
+                System.out.print(line);
+                System.out.println(spaces + "Here are the tasks in your list:");
+                for(int i = 0; i< taskList.size(); i++){
+                    System.out.print("    " + (i+1) + ".");
+                    taskList.get(i).printTask();
+                    System.out.println();
+                }
+                System.out.println(line);
+            } else if(command.contains("done")){
+                int taskIndex = Integer.parseInt(command.replaceAll("[^0-9]", ""));
+                taskIndex--;
+
+                taskList.get(taskIndex).markDone();
+                try{
+                    FileWriter fw = new FileWriter("data/tasks.txt");
+                    fw.write("");
+                    fw.close();
+                } catch (IOException e){
+                    System.out.println("Something went wrong" + e.getMessage());
+                }
+                for(Task task: taskList){
+                    file.writeFile(task);
+                }
+                System.out.println(line + "    Nice! I've marked this task as done:\n    "
+                        + taskList.get(taskIndex).getStatusIcon() + taskList.get(taskIndex).task );
+            } else if(command.contains("todo")|| command.contains("deadline")|| command.contains("event")){
```

Comment:
You should avoid complicated expression. Please refer to this https://nus-cs2113-ay2021s1.github.io/website/se-book-adapted/chapters/codeQuality.html